.github/labeler.yml

```diff
@@ -6,20 +6,16 @@ Cython / Python:
   - 'python/**'
```

Comment:
@JohnZed @dantegd is the label above `Cython / Python` correct?

Reply:
Traditionally we've grouped Cython / Python as a group together, since I don't think we have enough pure Cython PRs to guarantee its own label, so yes its correct unless we should match other RAPIDS repo labels.